src/js/letters/containers/Main.jsx

```diff
@@ -10,9 +12,32 @@ class Main extends React.Component {
   }
 
   render() {
+    let appContent;
+
+    if (this.props.lettersAvailability === 'available') {
+      appContent = this.props.children;
+    } else if (this.props.lettersAvailability === 'awaitingResponse') {
+      appContent = <LoadingIndicator message="Loading your letters..."/>;
+    } else if (this.props.lettersAvailability === 'unavailable') {
+      appContent = (
+        <div>
```

Comment:
I am guessing we will be using a warning like this in other circumstances, both for letters and for GIBS. We have these warnings  sprinkled throughout vets.gov code--I tried to enumerate some of them in this ticket: https://github.com/department-of-veterans-affairs/vets.gov-team/issues/3243#issuecomment-309521640

I have a secret desire to stop the proliferation of hard-coded messages like this and just have a single shared component that is used everywhere (possibly taking a prop that has the display name of the product, to customize the message). But for now it would be a start to just define this stuff in each app's helpers file.